src/main/docker/Dockerfile

```diff
@@ -16,15 +16,8 @@
 
 FROM cafapi/opensuse-base:2
 
-# Copy in python repo key
-COPY dev_python_repo_key.pub .
-
 # Update the OS packages and install Python
-RUN zypper addrepo https://download.opensuse.org/repositories/devel:/languages:/python:/Factory/openSUSE_Leap_15.1/devel:languages:python:Factory.repo && \
-    rpmkeys --import dev_python_repo_key.pub && \
-    rm dev_python_repo_key.pub && \
-    zypper -n refresh && \
-    zypper -n update && \
```

Comment:
Should we not keep the `zypper update` so it has the latest OS packages?

Reply:
Yes fair point.